asg.tf

```diff
@@ -87,10 +93,10 @@ resource "aws_autoscaling_group" "jenkins-master" {
   health_check_type         = "ELB"
   health_check_grace_period = "600"
 
-  tags = "${concat(var.asg_tags, list(map("key", "Name", "value", "${var.tags["Name"]}-jenkins-master", "propagate_at_launch", "true")))}"
+  tags = [ map("key", "Name", "value", "${var.tags["Name"]}-jenkins-master", "propagate_at_launch", "true") ]
```

Comment:
Did you mean to remove `var.asg_tags`?

Reply:
I removed cause using terraform 12 and I got some errors. My asg_tags were a list with one value, something like asg_tags = ["test"].

After some tests today I fixed using like that:

`    tags = concat(
    list(
      map("key", "Name", "value", "${var.tags["Name"]}-jenkins-master", "propagate_at_launch", "true"),
      map("key", "App", "value", "${jsonencode(var.asg_tags)}", "propagate_at_launch", "true")
    )
  )`